wsproto/utilities.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+wsproto/utilities
+~~~~~~~~~~~~~~~~~
+
+Utility functions that do not belong in a separate module.
+"""
```

Comment:
Not sure what the point is of splitting these out – they're not generic utilities that are useful in multiple places, but rather specific to HTTP header parsing, and thus are something that only `wsproto/connection.py` cares about.

Reply:
True.
These functions could be useful elsewhere - but mostly a personal preference what to do with such little helpers. Again, I introduced this when I blindly copied hyper-h2...